ClientManagementService/ClientManagementService.API/DTOMapper/PetDTOMapper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using ClientManagementService.API.DTO;
+using CorePet = ClientManagementService.Domain.Models.Pet;
+
+
+namespace ClientManagementService.API.DTOMapper
+{
+    public class PetDTOMapper
+    {
+        public static PetDTO ToDTOPet(CorePet corePet)

```

Comment:
May need to update our CorePet to have an Owner property and BreedName property. 

Then we need to update code to populate it.